pkg/commands/drift/command.go

```diff
@@ -191,14 +194,30 @@ func (c *DetectIamDriftCommand) Run(ctx context.Context, args []string) error {
 
 func driftMessage(drift *IAMDrift) string {
 	var msg strings.Builder
-	if len(drift.ClickOpsChanges) > 0 {
-		msg.WriteString(fmt.Sprintf("Found Click Ops Changes \n> %s", strings.Join(drift.ClickOpsChanges, "\n> ")))
-		if len(drift.MissingTerraformChanges) > 0 {
+	coKeys := maps.Keys(drift.ClickOpsChanges)
+	mtKeys := maps.Keys(drift.MissingTerraformChanges)
+	sort.Strings(coKeys)
+	sort.Strings(mtKeys)
+	if len(coKeys) > 0 {
+		msg.WriteString("Found Click Ops Changes\n")
```

Comment:
Can we update the wording a bit to clarify what these changes are? I.E. changes found that do not exist in terraform state, etc.

Reply:
Done